src/components/publishing/sections/image.tsx

```diff
@@ -14,12 +14,15 @@ interface ImageProps extends React.HTMLProps<HTMLDivElement> {
 }
 
 const Image: React.SFC<ImageProps> = props => {
-  const { image, layout, width, height } = props
+  const { image, layout, width, height, children } = props
+  const child = children && children
```

Comment:
What does the `&& children` do here?

Reply:
works similar to `children ? children : false`

Reply:
I could be wrong here, but I think then it'll be fine to just pass the children in directly - if not, then merge away 👍

Reply:
The caption component defaults to showing children over a passed caption, so we do want to pass false here rather than children in this case.